compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangLetExpression.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.wso2.ballerinalang.compiler.tree.expressions;
+
+import org.ballerinalang.model.tree.NodeKind;
+import org.ballerinalang.model.tree.expressions.LetExpressionNode;
+import org.wso2.ballerinalang.compiler.semantics.model.SymbolEnv;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeVisitor;
+import org.wso2.ballerinalang.compiler.tree.types.BLangLetVariable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This represents a let expression in ballerina.
+ *
+ * @since 1.2.0
+ */
+public class BLangLetExpression extends BLangExpression implements LetExpressionNode {
+
+    public List<BLangLetVariable> letVarDeclarations = new ArrayList<>();
+    public BLangExpression expr;
+    public SymbolEnv env;
+
+    @Override
+    public void accept(BLangNodeVisitor visitor) {
+        visitor.visit(this);
+    }
+
+    @Override
+    public NodeKind getKind() {
+        return NodeKind.LET_EXPR;
+    }
```

Comment:
Shall we override the toString() method as well?

Reply:
Fixed.